firmware/Core/Src/Controller.c

```diff
@@ -103,9 +103,43 @@ static Controller_State_t LogData_State(void)
     return CTRL_DO_MATH;
 }
 
+float savedTemps[];
+float loopCounter;
+
+
 static Controller_State_t DoMath_State(void)
 {
-    return CTRL_ACTUATE_FRIDGE;
+    float temps[5] = {t1, t2, t3, t4, t5}; //5 temp probe readings
+    float temp_diffs[5]; //to store deltas between probe readings and avg temp
+    float avgTemp = 0; //mean temperature
```

Comment:
Is this the temp we're shooting for? 0? I though Astra zenica was like 5 or 6.

Reply:
I just initialized it to 0 but it will maintain between 1C and 3C, in our abstract I think we said 2C